app/assets/stylesheets/quby/single_page/layout.css.scss

```diff
@@ -0,0 +1,4 @@
+.single_page {
```

Comment:
Doch, is het wel lastig raden als je - en _ door elkaar gebruikt, bij overal in quby gebruiken we -, dus laten we dat aanhouden.